cliStocksTracker.py

```diff
@@ -298,36 +245,37 @@ def average_buyin(self, buys: list, sells: list):
 
         return count, bought_at
 
+    # download all ticker data in a single request
+    # harder to parse but this provides a signficant performance boost
+    def download_market_data(self, args, stocks):
+        # get graph time interval and period
+        time_period = args.time_period if args.time_period else "1d"
+        time_interval = args.time_interval if args.time_interval else "1m"
+
+        try:
+            return market.download(
+                tickers=stocks,
+                period=time_period,
+                interval=time_interval,
+                progress=False,
+            )
+        except:
```

Comment:
I would love to see a specific exception here - just so if something else goes wrong the user still gets a stack trace for forward our way.

Reply:
this was merged from the main branch - i've raised the exception explicitly. if you have something else in mind could you do it as a separate PR?

Reply:
I'm not going to freak out about it - we'll just fix it later.
This does work! Whats there to complain about?